skimage/draw/draw.py

```diff
@@ -125,7 +125,7 @@ def circle(r, c, radius, shape=None):
     return ellipse(r, c, radius, radius, shape)
 
 
-def set_color(img, coords, color):
+def set_color(img, coords, color, alpha=None):
```

Comment:
It's a common pattern we use in a lot of places that makes it possible to change the default later on.  Imagine you have `alpha=1` here, and later decide you want `alpha=0.5` as the default.  When you then get "alpha=1", you never know if the user went for the default value or for 1.  With ``None``, that is explicit, so you can do the check ``if alpha is None: warn the user that the default has changed``.